tools/wtdbg/macros.xml

```diff
@@ -0,0 +1,22 @@
+<macros>
+    <xml name="requirements">
+        <requirements>
+        <requirement type="package" version="1.2.8.1">wtdbg</requirement>
+            <yield/>
+        </requirements>
+    </xml>
+    <xml name="citations">
+        <citations>
+            <citation type="bibtex">
+@misc{githubwtdbg-1.2.8,
```

Comment:
wouldn't it be better to cite the original repo?

Reply:
This is the orig one! :)

Reply:
I was referring to this one: https://github.com/ruanjue/wtdbg

Reply:
I know, but the other one is really "main" one. More help, recent updates, detailed description etc. Also the author has asked to refer githubwtdbg-1.2.8 in the readme.